new-site/src/docs/elements/components/date-input/customisation.mdx

```diff
@@ -19,13 +19,18 @@ You can use the `style` property to customise the component. See all available t
 ### Customisation example
 <Playground>
 
-
 ```jsx
-
-```
-
-```html
-
+<div style={{maxWidth: '400px'}}>
+  <DateInput
```

Comment:
This seems be the same as in Code tab? We probably need to explain better above how date input can be customised, and then use those props here.

Reply:
Yes. I tried to fix the customisation examples and add some working placeholders. But I guess some components might not need customisation tab at all.